packages/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx

```diff
@@ -77,13 +84,19 @@ export class DropdownToggleCheckbox extends React.Component<DropdownToggleCheckb
       /* eslint-enable @typescript-eslint/no-unused-vars */
       ...props
     } = this.props;
+
+    const spinner = <Spinner diameter="1em" isSVG aria-label={defaultProgressAriaLabel} />;
```

Comment:
My Voice Over is never reading this aria label - I'm wondering if the label needs an aria-live attribute. From in browser testing, it seems like that works. Does that sounds right, @thatblindgeye

Reply:
>My Voice Over is never reading this aria label - I'm wondering if the label needs an aria-live attribute. From in browser testing, it seems like that works. **Does that sounds right**

@jenny-s51  @nicolethoen Ah yeah it does!

Kind of related to that, but when testing with an aria-live added into the spinner, "Loading..." gets announced twice (once for the aria-label passed into defaultProgressAriaLabel, and another time from the Spinner's aria-valuetext prop. It might be worth adding another prop for aria-valuetext that gets passed into Spinner that acts as the "loading" verbiage, and reserving defaultProgressAriaLabel to describe what's being loaded (per Spinner's aria-label prop description).